api/server/router/local/local.go

```diff
@@ -0,0 +1,161 @@
+package local
```

Comment:
What does "local" mean here? default router? core routes?  local to.. the daemon?

Sorry if this should be obvious, I'm still familiarizing myself with the engine code base.  A more descriptive package name might help clarify.